modules/mcaptcha.nix

```diff
@@ -0,0 +1,279 @@
+{
+  config,
+  lib,
+  options,
+  pkgs,
+  ...
+}:
+with builtins;
```

Comment:
When PRing against nixpkgs I got the comment that `with builtins;` should be avoided for such a large scope (=rest of the file, basically). Builtins are used rarely, so you can probably spell it out.